be/src/exec/pipeline/pipeline_driver_queue.cpp

```diff
@@ -44,7 +44,12 @@ DriverPtr QuerySharedDriverQueue::take(size_t* queue_index) {
                 break;
             }
             _is_empty = true;
-            _cv.wait(lock);
+            // wait for drivers' arrival, but awake execution threads to process dying QueryContexts after
+            // 100 micro-seconds' waiting.
+            auto cv_status = _cv.wait_for(lock, std::chrono::microseconds(100));
```

Comment:
This is reasonable？

Reply:
wake up and do clean job? 
I think it's fine to wake up till next pipeline arrive? (maybe unused objects will be last for a long time)